datastore/protobuf/datastore.proto

```diff
@@ -127,9 +133,29 @@ message GetObsRequest {
   // TODO: add search filters for other metadata
 }
 
-
 message GetObsResponse {
   int32 status = 1;
   string error = 2; // any error description (empty on success)
   repeated Metadata2 observations = 3;
 }
+
+//---------------------------------------------------------------------------
+
+message TSMdataGroup {
+  repeated TSMetadata combos = 1;
+}
+
+message GetTSACRequest {
+  repeated string attrs = 1; // one or more TSMetadata field names
+  bool include_instances = 2; // whether to include all matching time series instances in response
```

Comment:
For the `include_instances=true` case, I wonder if this is really what we want. It will now always return all time-series metadata, just grouped and ordered in a different way. That is quite a heavy call. It is not so clear to me what issue #67 will actually be used for.

Reply:
Issue #67 was requested by Amund. Maybe you can check with him if he really needs it after all? I did notify both Amund and Johannes about this new feature (i.e. the updated PR) after I added it on Friday, but I haven't heard anything from them at the time of writing. Generally though, I guess a heavy call could be justified if it's not made too often.